src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -22,24 +16,28 @@
 import org.jabref.logic.importer.ImportException;
 import org.jabref.logic.importer.ImportFormatReader;
 import org.jabref.logic.importer.ImportFormatReader.UnknownFormatImport;
+import org.jabref.logic.importer.ParseException;
 import org.jabref.logic.importer.fetcher.DoiFetcher;
+import org.jabref.logic.importer.fileformat.BibtexParser;
 import org.jabref.model.database.BibDatabaseMode;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.identifier.DOI;
+import org.jabref.model.util.OptionalUtil;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ClipBoardManager implements ClipboardOwner {
+public class ClipBoardManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ClipBoardManager.class);
+    public static final DataFormat XML = new DataFormat("application/xml");
```

Comment:
Is this used anywhere?

Reply:
Yes, when we put xml in the clipboard (which may happen for copied citation styles)

Reply:
Yes, when we put xml in the clipboard (which may happen for copied citation styles)

Reply:
We have already a class DragAndDropDataFormats, maybe you could rename that and add it to the collection?

Reply:
I don't want to interfere with your PR (especially, since I think it might completely remove the `DragAndDropFormats` class or make it internal to the `DragAndDropManager`).